tests/common.go

```diff
@@ -4711,6 +4717,31 @@ func TeardownForTestcase(contexts []*scheduler.Context, providers []string, Clou
 	return true
 }
 
+func updatePxRuntimeOpts() error {
+	if pxRuntimeOpts != "" {
+		dash.Infof("Setting run time options: %s", pxRuntimeOpts)
+		optionsMap := make(map[string]string)
+		runtimeOpts, err := splitCsv(pxRuntimeOpts)
+		if err != nil {
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Error parsing run time options, err : %v", err))
+		}
+
+		for _, opt := range runtimeOpts {
+			if !strings.Contains(opt, "=") {
+				dash.VerifyFatal(false, true, fmt.Sprintf("Given run time option is not in expected format key=val, Actual : %v", opt))
+			}
+			optArr := strings.Split(opt, "=")
+			optionsMap[optArr[0]] = optArr[1]
+		}
+		currNode := node.GetWorkerNodes()[0]
+		err = Inst().V.SetClusterRunTimeOpts(currNode, optionsMap)
```

Comment:
You could just `return Inst().V.SetClusterRunTimeOpts(currNode, optionsMap)`

Reply:
fixed